src/ejabberd_piefxis.erl

```diff
@@ -55,7 +55,7 @@
 %%%==================================
 %%%% Import file
 -define(NS_PIE, <<"urn:xmpp:pie:0">>).
--define(NS_PIEFXIS, <<"http://www.xmpp.org/extensions/xep-0227.html#ns">>).
+-define(NS_PIEFXIS, <<"https://xmpp.org/extensions/xep-0227.html#ns">>).
```

Comment:
No, the namespace is http://www.xmpp.org/extensions/xep-0227.html#ns and you can't change it just because you want to.